### PR TITLE
Rustfmt and clippy

### PR DESCRIPTION
**Pull request by ustulation**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/crust/470)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @vinipsmaker

(maidsafe_highfive has picked a reviewer for you, use r? to override)

**Comment by vinipsmaker** [comment]
CI tests passed. Gonna take a look.

File: src/beacon.rs [hunk 1]
```diff
@@ -213,54 +226,60 @@ pub fn seek_peers(port: u16, guid_to_avoid: Option<GUID>) -> Result<Vec<SocketAd
...
+                                                   }
+                                               }
+                                               _ => {
+                                                   // The response is invalid
+                                                   continue;
+                                               }
+                                           };
+                                       }
+                                       Ok(())
+                                   });
```
Comment (inetic) [inline src/beacon.rs]: I can't see how this is more readable to anyone (apart from clippy or rustfmt).

**Pull request merged**